transport/http/constants.go

```diff
@@ -20,8 +20,12 @@
 
 package http
 
+import "time"
+
 const transportName = "http"
 
+var defaultConnTimeout = 500 * time.Millisecond
```

Comment:
I have no recollection of this... >.> (honestly I'm not aware of the style guide on this cc @abhinav)